crates/cheatnet/tests/cheatcodes/store.rs

```diff
@@ -0,0 +1,208 @@
+use crate::assert_success;
+use crate::common::state::{create_cached_state, create_cheatnet_state};
+use crate::common::{felt_selector_from_name, get_contracts};
+use blockifier::abi::abi_utils::starknet_keccak;
+use cairo_felt::Felt252;
+use cheatnet::runtime_extensions::call_to_blockifier_runtime_extension::rpc::call_contract;
+use cheatnet::runtime_extensions::forge_runtime_extension::cheatcodes::deploy::deploy;
+use cheatnet::runtime_extensions::forge_runtime_extension::cheatcodes::load::calculate_variable_address;
+use cheatnet::runtime_extensions::forge_runtime_extension::cheatcodes::store::store;
+use conversions::felt252::FromShortString;
+
+#[test]
+fn store_simple_state() {
+    let mut cached_state = create_cached_state();
+    let (mut blockifier_state, mut cheatnet_state) = create_cheatnet_state(&mut cached_state);
+
+    let contract = Felt252::from_short_string("HelloStarknet").unwrap();
+    let contracts = get_contracts();
+
+    let class_hash = blockifier_state.declare(&contract, &contracts).unwrap();
+
+    let contract_address = deploy(&mut blockifier_state, &mut cheatnet_state, &class_hash, &[])
+        .unwrap()
+        .contract_address;
+
+    store(
+        &mut blockifier_state,
+        contract_address,
+        &starknet_keccak("balance".as_ref()),
```

Comment:
Could this be `felt_selector_from_name`?

Reply:
good catch :)